src/main/java/org/springframework/data/couchbase/core/ExecutableFindByIdOperationSupport.java

```diff
@@ -53,7 +54,12 @@
 
 		@Override
 		public T one(final String id) {
-			return reactiveSupport.one(id).block();
+			try {
+				return reactiveSupport.one(id).block();
+			} catch(DocumentNotFoundException dnfe){
+				dnfe.printStackTrace();
```

Comment:
this is a leftover I assume?

Reply:
I think it needs to be here in addition to ReactiveFindByQueryOperationSupport.  I had myself convinced there are different code-paths.  I'll check again.

Reply:
Sorry, I meant the printStackTrace will dump into the logs and seems to be a leftover of debugging